Feature/landing page html

benitaisaac opened the pull request:
static html landing page to render to handlebars.

benitaisaac committed:
Added stock hero image formatted with bootstrap

benitaisaac committed:
added stock images and changed hero image on landing page

benitaisaac committed:
added images of communities and updated communities div in landing page

benitaisaac committed:
changed footer for landing page

benitaisaac committed:
added a todo comment for communities section

benitaisaac committed:
pulled new login files and edited static landing page images for responsiveness

Mwrightvet reviewed (changes requested):
The landing page.html should be moved to the homepage.handlebars file.

benitaisaac committed:
Merge branch 'main' of https://github.com/kerilsen/share-sphere into feature/landing-page-html

benitaisaac committed:
added code to homepage.handlebars, but not formatted correctly. Looked at header of main.handlebars

benitaisaac commented:
Made the changes

benitaisaac commented:
Moved static html code to homepage.handlebars although it probably won't render because it isn't formatted properly. I'm keeping the static landing-page.html file as a reference in case we need it while refactoring html to handlebars.  
I also added my code from the landing-page.html file head into the main.handlebars file although it looked like the code was already correct.
